### PR TITLE
RHEL release

### PR DESCRIPTION
**Pull request by surajsbharadwaj**
### Description

Please include a summary of the change and which issue is fixed. Also, please include the motivation and context for the change. List any dependencies that are required for this change.

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [ ] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

**Review by Beilmann — DISMISSED** [review]

File: submodules/power_sap_instance_init/scripts/init_powervs.sh [hunk 1]
```diff
@@ -179,20 +179,89 @@ fi
...
+        if [[ "$ARCH" == "x86_64" ]]; then
+      #### Wait for registration to complete
+ 	     count=0
+             while [[ $count -le 15 ]]; do
+                sleep 60
+                OS_Activated="$(subscription-manager status | grep -c "Overall Status: Current")"
+                if  [[ "$OS_Activated" -ge 1 ]] ; then
+        	        echo "OS is Registered"
+    		          break;
+	              fi
```
Comment (ocofaigh) [inline submodules/power_sap_instance_init/scripts/init_powervs.sh:211]: can you fix the indentation here so it is consistent across the script?

File: submodules/power_sap_instance_init/scripts/init_powervs.sh [hunk 2]
```diff
@@ -179,20 +179,89 @@ fi
...
+            cmd=$(grep /usr/local/bin/rhel-cloud-init.sh < /usr/share/powervs-fls/powervs-fls-readme.md | grep Private.proxy.IP.address | sed "s/Private.proxy.IP.address:3128/$proxy_ip_and_port/g"); $cmd
+            count=1
+	         while [[ $count -le 15 ]]
+            do
+                sleep 60
+                count=$(( count + 1 ))
+                if grep -i failed  /var/log/powervs-fls.log; then
+ 		              echo "RHEL registration has failed, exiting"
+		              exit 1
+	            	fi
```
Comment (ocofaigh) [inline submodules/power_sap_instance_init/scripts/init_powervs.sh:229]: can you fix the indentation here so it is consistent across the script?

File: submodules/power_sap_instance_init/scripts/init_powervs.sh [hunk 3]
```diff
@@ -179,20 +179,89 @@ fi
...
+      ##### Check if registration was successful
+                #get the subscription server name from /var/log/powervs-fls.log
+                subscription_server=$(grep sap_hana /usr/share/powervs-fls/powervs-fls-readme.md | awk -F"-u" '{ print $NF }' | awk '{ print $1 }')
+		            if subscription-manager config | grep "$subscription_server"; then
```
Comment (ocofaigh) [inline submodules/power_sap_instance_init/scripts/init_powervs.sh:233]: can you fix the indentation here so it is consistent across the script?

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@surajsbharadwaj can you fix the indentation in `init_powervs.sh` so it is consistent across the script. You should set your IDE up so it will handle the indentation for you.

File: submodules/power_sap_instance_init/scripts/init_powervs.sh (hunk 1 above)
Comment (surajsbharadwaj) [reply submodules/power_sap_instance_init/scripts/init_powervs.sh]: Done. Added shell extension. Thank you :)

**Comment by ocofaigh** [comment]
Thanks @surajsbharadwaj looks better. Should this be a minor version release? What release notes do you want me to include as part of it?

**Review by surajsbharadwaj — COMMENTED** [review]
Looks okay

**Comment by surajsbharadwaj** [comment]
Rhel support for Powervs init
Readmes for submodules

**Pull request merged**